code/modules/events/event_dynamic.dm

```diff
@@ -99,6 +99,8 @@ var/list/event_last_fired = list()
 	//and start working via the constructor.
 	new picked_event
 
+	score["eventsendured"]++
```

Comment:
I could always change it to less powerful terminology like "Random Events this Shift", but it'll be fine for the most part, plus it counts positively